engines/mohawk/myst_stacks/myst.cpp

```diff
@@ -2292,6 +2293,38 @@ uint16 Myst::rocketSliderGetSound(uint16 pos) {
 	return (uint16)(9530 + (pos - 216) * 35.0 / 61.0);
 }
 
+
+uint16 Myst::rocketCheckIfSoundMatches(uint16 sound1, uint16 sound2) {
+
+	bool fuzzylogic = false;
+
+  	if (ConfMan.getBool("fuzzy_logic")) {
+		fuzzylogic = true;
+	}
```

Comment:
You can either do:

    const bool fuzzyLogic = ConfMan.getBool("fuzzy_logic");

Or remove the fuzzyLogic variable altogether, since there's only a single place where the option is checked:

    if (!ConfMan.getBool("fuzzy_logic")) {
        ....